### DISCUSSION TITLE
Bài 7

### DISCUSSION THREAD

**Discussion by phucthinhb3** [phucthinhb3/Lop-11A03-Nhom-07#10] [General]
Có bài mới rồi nha! Mọi người lên notion để lấy file bài về nhá giờ thì tao phân công việc cho
-Thịnh, Tuấn Anh, Phong, Tín: Ý nghĩa code (từng bước hoạt động) của bài tập 2, 3, 4 về Serial Monitor trên Activitiy 
-Khôi: So sánh hàm Serial.print() và Serial.write()

**Comment by tintruongvinh** [comment]
Ok nha

**Comment by Minh-Khoi16** [comment]
k
